### DISCUSSION TITLE
How does one set a password in seeds.ts?

### DISCUSSION THREAD

**Discussion by esemeniuc** [blitz-js/blitz#2630] [Q&A]
I understand that the password uses a secret environment variable, however between 2 password resets, the hashed password is different. I tried to base64 decode it, and there is some binary data which I cannot understand. Can someone explain how can I set  my password in seeds.ts?

**Comment by flybayer** [comment]
You can do it like this:

```ts
  import {SecurePassword} from 'blitz'
  import db from 'db'

  const hashedPassword = await SecurePassword.hash(password)
  const user = await db.user.create({
    data: {email: 'user@example.com', hashedPassword},
  })
```

the hashedPassword is different on each run, even for the same password, for security reasons.

See the `login` mutation for how to verify a hashed password

**Reply by esemeniuc** [reply]
Thank you Brandon :)
